fonduer/parser/visual_linker.py

```diff
@@ -45,13 +45,21 @@ def parse_visual(self, document_name, sentences, pdf_path):
             yield sentence
 
     def extract_pdf_words(self):
+        self.logger.debug(
+            "pdfinfo '{}' | grep -a Pages | sed 's/[^0-9]*//'".format(self.pdf_file)
+        )
```

Comment:
This way, we can switch to debug level logging and get an exact log of the command that was run.